node/src/GlideClusterClient.ts

```diff
@@ -898,17 +888,15 @@ export class GlideClusterClient extends BaseClient {
      * Returns information about the function that's currently running and information about the
      * available execution engines.
      *
-     * See https://valkey.io/commands/function-stats/ for details.
-     *
-     * Since Valkey version 7.0.0.
+     * @see {@link https://valkey.io/commands/function-stats/|valkey.io} for details.
+     * @remarks Since Valkey version 7.0.0.
      *
      * @param route - The client will route the command to the nodes defined by `route`.
      *     If not defined, the command will be routed to all primary nodes.
      * @returns A `Record` with two keys:
      *     - `"running_script"` with information about the running script.
      *     - `"engines"` with information about available engines and their stats.
-     *
-     * See example for more details.
+     *     - See example for more details.
```

Comment:
It is not a key of the record, so I put it outside the list

Reply:
If you don't add it to the list, it doesn't get rendered.  Oooh, maybe I can add a second @returns tag... that might work

Reply:
It renders fine for me:
![image](https://github.com/user-attachments/assets/41e433f5-0cb9-4392-a5b6-686567ef3fc8)
I put a blank line there, because it is rendered as a linebreak (aka `<br>`).